security/tailscale/LICENSE

```diff
@@ -0,0 +1,21 @@
+BSD 2 Clause License
```

Comment:
Don't really need this, your copyright gets folded into LICENSE page in root directory.. but from every page, not just this unattached one... and the BSD is standard amnyway :)